lib/hyper-balance/src/lib.rs

```diff
@@ -38,15 +36,15 @@ where
     type Output = http::Response<PendingUntilFirstDataBody<T, B>>;
 
     fn instrument(&self, handle: T, rsp: http::Response<B>) -> Self::Output {
-        let (parts, body) = rsp.into_parts();
-        let handle = if body.is_end_stream() {
-            drop(handle);
-            None
-        } else {
-            Some(handle)
-        };
-        let body = PendingUntilFirstDataBody { handle, body };
-        http::Response::from_parts(parts, body)
+        rsp.map(move |body| {
```

Comment:
purely out of interest: is the change from `into/from_parts` to `map` related to the change to Rust 2018?

Reply:
Nah, something I just noticed when looking through the file 🤷‍♂ .